src/pages/form-validation/sectioned-form/index.tsx

```diff
@@ -0,0 +1,496 @@
+        },
+    },
+    submitButtonContainer: {
+        marginTop: theme.spacing(5),
+        display: 'flex',
+        justifyContent: 'flex-end',
+        [theme.breakpoints.down('xs')]: {
+            width: '100%',
+        },
+    },
+    submitButton: {
+        [theme.breakpoints.down('xs')]: {
+            width: '100%',
+        },
+    },
+    zipInput: {
+        marginRight: theme.spacing(2),
+        marginLeft: theme.spacing(2),
+        [theme.breakpoints.down('xs')]: {
+            margin: `${theme.spacing(mobileInputMarginSpacing)}px 0 ${theme.spacing(mobileInputMarginSpacing)}px 0`,
+        },
+    },
+    textField: {
+        height: 72,
+        width: '100%',
+    },
+}));
+
+export const SectionedFormValidation = (): JSX.Element => {
+    const theme = useTheme();
+    const classes = useStyles(theme);
+    const dispatch = useDispatch();
+    const [name, setName] = useState('');
+    const [level, setLevel] = useState('level II');
+    const [address, setAddress] = useState('');
+    const [addressLine2, setAddressLine2] = useState('');
+    const [city, setCity] = useState('');
+    const [state, setState] = useState('');
+    const [firstName, setFirstName] = useState('');
+    const [lastName, setLastName] = useState('');
+    const [email, setEmail] = useState('');
+    const [zip, setZip] = useState('');
+    const [emailError, setEmailError] = useState('');
+    const [showRequiredError, setShowRequiredError] = useState(false);
+    const MAX_CHARS_LIMIT = 50;
+    const pxbProtection = 'PXB Protection';
+    const pxbProtectionDescription = 'PXB Protection provides a three-year power expert warranty.';
+
+    const emailRegex = new RegExp(/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i);
+
+    const validateEmail = useCallback(
+        (value: string): void => {
+            const tempEmail = value;
+            let tempEmailError = '';
+            if (!tempEmail.trim()) {
+                tempEmailError = 'Required';
+            } else if (!emailRegex.test(tempEmail)) {
+                tempEmailError = 'Please enter a valid email address';
+            }
+            setEmailError(tempEmailError);
+        },
+        [setEmailError]
+    );
+
+    const onEmailBlur = useCallback((): void => {
+        validateEmail(email);
+    }, [validateEmail, email]);
+
+    const onEmailChange: OnChangeHandler = useCallback(
+        (event) => {
+            setEmail(event.target.value);
+            if (emailError) {
+                validateEmail(event.target.value);
+            } else {
+                setEmailError('');
+            }
+        },
+        [setEmail, emailError, validateEmail, setEmailError]
+    );
+
+    const characterLimitsHelperText = (
+        <>
+            <span>{!name && showRequiredError ? 'Required' : 'For example, Facility or Campus name'}</span>
+            <span style={{ float: 'right' }}>{`${name.length}/${MAX_CHARS_LIMIT}`}</span>
```

Comment:
Can we make this stay theme.palette.text.secondary, even on error? Like this:

![image](https://user-images.githubusercontent.com/8997218/131366296-d8f917fd-ea79-40cf-970f-c616704ca269.png)